cmd/agent/app/options/options.go

```diff
@@ -112,4 +117,5 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.IntVar(&o.ConcurrentClusterSyncs, "concurrent-cluster-syncs", 5, "The number of Clusters that are allowed to sync concurrently.")
 	fs.IntVar(&o.ConcurrentWorkSyncs, "concurrent-work-syncs", 5, "The number of Works that are allowed to sync concurrently.")
 	o.RateLimiterOpts.AddFlags(fs)
+	fs.StringVar(&o.MetricsBindAddress, "metrics-bind-address", ":8080", "The TCP address that the controller should bind to for serving prometheus metrics(e.g. 127.0.0.1:8088, :8088)")
```

Comment:
Karmada-controller-manager already has `MetricsBindAddress`.  It's ok to add it in agent.  How about move the flag above  `o.RateLimiterOpts.AddFlags(fs)`?

Reply:
>How about move the flag above o.RateLimiterOpts.AddFlags(fs)?

Check again please

Reply:
For me /lgtm
/cc @RainbowMango